Sources/SwiftKuery/Column.swift

```diff
@@ -59,18 +77,30 @@ public class Column: Field, IndexColumn {
         return _table
     }
     
-    /// Initialize an instance of Column.
-    ///
+    // MARK: Column Initializer
+    
+    /**
+     The initializer for the `Column` class. This creates an instance of `Column` using the provided parameters.
+     Name must be provided, but all other fields will default to either nil or false if not given.
+     ### Usage Example: ###
+     In this example, an instance of the `Column` class is created to match The person_id column of an SQL table.
+     The person_id column in the database is stored as an Int32 and is the table's primary key.
+     To represent this a `Column` is initialised with name set to "person_id", type set as Int32.self (self is required to pass Int32 as the class) and primaryKey set to true.
+     A feature of the primary key is that they must be unique can cannot be null so unique and notNull are set to true.
+     ```swift
+     let person_id = Column("person_id", Int32.self, autoIncrement: true, primaryKey: true, notNull: true, unique: true)
+     ```
+     */
     /// - Parameter name: The name of the column.
-    /// - Parameter type: The type of the column.
-    /// - Parameter length: The length of the column values according to the type.
-    /// - Parameter autoIncrement: An indication whether the column autoincrements.
-    /// - Parameter primaryKey: An indication whether the column is the primary key of the table.
-    /// - Parameter notNull: An indication whether the column is not nullable.
-    /// - Parameter unique: An indication whether the column values have to be unique.
-    /// - Parameter defaultValue: The default value of the column.
-    /// - Parameter check: The expression to check for values inserted into of the column.
-    /// - Parameter collate: The collation rule for the column.
+    /// - Parameter type: The type of the column. Defaults to nil.
+    /// - Parameter length: The length of the column values according to the type. Defaults to nil.
+    /// - Parameter autoIncrement: An indication whether the column autoincrements. Defaults to false.
+    /// - Parameter primaryKey: An indication whether the column is the primary key of the table. Defaults to false.
+    /// - Parameter notNull: An indication whether the column is not nullable. Defaults to false.
```

Comment:
I found this confusing - please can you reword to 
"Indicates whether the column has the NOT NULL attribute. Defaults to false, which permits null values."